avalon_listener/listener_config.toml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# --------------------------------------------------
+# Logging -- configuration of service logging
+# --------------------------------------------------
+[Logging]
+LogLevel = "INFO"
+LogFile  = "__screen__"
+
+# -------------------------------------------------------------------
+# Listener -- configuration of Avalon Listener
+# -------------------------------------------------------------------
+
+[Listener]
+listener_port = 1947
+max_work_order_count = 10
+
+[KvStorage]
+remote_url = "http://avalon-lmdb:9090"
```

Comment:
remote_url should be changed to some meaningful name,

Reply:
ok